lib/src/models/multireddit.dart

```diff
@@ -241,8 +244,23 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
           .replaceAll(_multiredditRegExp, name)
           .replaceAll(_userRegExp, user);
 
+  // Converts input text into camelCase.
+  static String _convertToCamelCase(String str) {
+    if (str == null) {
+      return null;
+    }
+    final RegExp _invalidRegExp = new RegExp(r'(\s|\W|_)+');
+    final splitStrList = str.split(_invalidRegExp);
+    // https://giphy.com/gifs/doctor-who-david-tennant-XTJACShBsxPws
+    splitStrList[0] += splitStrList.sublist(1)?.fold(
+        '',
+        (prevValue, next) =>
+            prevValue + next.substring(0, 1).toUpperCase() + next.substring(1));
```

Comment:
```dart
next.substring(0, 1).toUpperCase() + next.substring(1))
``` 
This bugs me a bit (not your fault), but I don't think there's a better way :(

Reply:
I refer you to the gif :p